ScratchDjango/User/views.py

```diff
@@ -93,9 +133,13 @@ def login_template(request):
 
         response = requests.request("POST", url, headers=headers, data=payload)
         if response.ok:
-            jwt_token = response.json()["access"]
-            messages.success(request, "Logged In")
-            return redirect("check_login_template", token=jwt_token)
+            data = response.json()
+            if data["status"] == GOOGLE_AUTHENTICATOR or data["status"] == "email":
+                return render(request, "otp.html", {"email": email})
+            else:
+                jwt_token = response.json()["access"]
+                messages.success(request, "Logged In")
+                return redirect("check_login_template", token=jwt_token)
```

Comment:
Why we are not using render here?

Reply:
Because check_login_template is a view that will render the success.html and failure.html pages if the token in valid or not.

Reply:
Are these templates somehow being used in react?

Reply:
No, these were only used before we moved the frontend to react